argus/action_manager/windows.py

```diff
@@ -306,9 +308,47 @@ def __init__(self, client, config, os_type=util.WINDOWS_SERVER_2016):
 
 
 class WindowsNanoActionManager(WindowsSever2016ActionManager):
+    _DOWNLOAD_SCRIPT = "FastWebRequest.ps1"
+    _COMMON = "common.psm1"
+    _RESOURCE_DIRECTORY = r"C:\nano_server"
+
     def __init__(self, client, config, os_type=util.WINDOWS_NANO):
         super(WindowsNanoActionManager, self).__init__(client, config, os_type)
 
+    def specific_prepare(self):
+        """Prepare resources For Windows NanoServer."""
+        super(WindowsNanoActionManager, self).specific_prepare()
+
+        if not self.is_dir(self._RESOURCE_DIRECTORY):
+            self.mkdir(self._RESOURCE_DIRECTORY)
+
+        LOG.info("Copy common script for Windows Nanoserver.")
+        resource_path = os.path.join(
+            os.path.abspath(os.path.dirname(__file__)),
+            "..", "resources", "windows", "nano_server",
+            self._COMMON)
+        resource_path = os.path.normpath(resource_path)
+        instance_path = ntpath.join(self._RESOURCE_DIRECTORY, self._COMMON)
+        self._client.copy_file(resource_path, instance_path)
+
+        LOG.info("Copy Download script for Windows Nanoserver.")
+        resource_path = os.path.join(
```

Comment:
Redundant code block. Why don't you put that code block is some method (`get_resource_path` for example) ?